nanowallet/src/start.html

```diff
@@ -9,6 +9,7 @@
     <link rel="stylesheet" href="css/ngToast.min.css">
     <link rel="stylesheet" href="css/NEMonster.css">
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Noto+Sans:100,400,700,900">
+    <script src="https://connect.trezor.io/9/trezor-connect.js"></script>
```

Comment:
Is there a reason to use the script vs package(https://www.npmjs.com/package/@trezor/connect)?